test/integration/apiserver/admissionwebhook/validatingadmissionpolicy_test.go

```diff
@@ -0,0 +1,716 @@
+package admissionwebhook
```

Comment:
This test shares a lot of code with the similar test for admission webhook. Not sure if we want to colocate them, duplicate the code, or factor something out to another package